clients/src/main/java/org/apache/kafka/clients/admin/QuorumInfo.java

```diff
@@ -94,21 +104,24 @@ public String toString() {
 
     public static class ReplicaState {
         private final int replicaId;
+        private final Optional<Uuid> replicaDirectoryId;
         private final long logEndOffset;
         private final OptionalLong lastFetchTimestamp;
         private final OptionalLong lastCaughtUpTimestamp;
 
         ReplicaState() {
-            this(0, 0, OptionalLong.empty(), OptionalLong.empty());
+            this(0, Optional.of(Uuid.ZERO_UUID), 0, OptionalLong.empty(), OptionalLong.empty());
```

Comment:
So, this should be Optional.empty() I think.

Reply:
Fixed.